ui/src/app/shared/components/suspense-monaco-editor.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react';
+import {MonacoEditorProps} from 'react-monaco-editor';
+import MonacoEditor from 'react-monaco-editor';
```

Comment:
```suggestion
import type {MonacoEditorProps} from 'react-monaco-editor';
import type MonacoEditor from 'react-monaco-editor';
```

I originally made these explicit `import type`s, but that made `tslint` error out as it doesn't understand the syntax. Need to migrate to ESLint (as `tslint` has long been deprecated)

Reply:
Migrated to ESLint in #12163. As mentioned there, there's still a syntax reading error due to the old version of Prettier. But #12163 unblocks upgrading Prettier, which should be a lot more straightforward.

And _then_ we can start using `import type` 😅

Reply:
Fixed in #12514